Snippets/Gateway/Gateway_2/Channels/ProvideConfiguration.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace Gateway_2.Channels
+﻿namespace Channels
```

Comment:
why changing something on the v2 snippet?

Reply:
I removed the Gateway_3 prefix so also applied it to the other snippets. After reading the docs README I don't think these are used to identify any package and can safely be aligned